source/default_mode/Host.h

```diff
@@ -449,17 +449,20 @@ class Host: public Organism {
   /**
    * Input: The pointer to the organism that is to be added to the host's symbionts.
    *
-   * Output: None
+   * Output: The int describing the symbiont's position ID, or 0 if it did not successfully
+   * get added to the host's list of symbionts.

```

Comment:
Can syms not be at position 0? If so, should be specified since that's different than normal